GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiedAccountDetailResult.h

```diff
@@ -25,19 +25,18 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @class GIDVerifiableAccountDetail;
+@class GIDToken;
 @class OIDAuthState;
 @class OIDTokenResponse;
 
 /// A helper object that contains the result of a verification flow.
 /// This will pass back the necessary tokens to the requesting party.
 @interface GIDVerifiedAccountDetailResult : NSObject <GIDVerifiedAccountDetailHandling>
 
-/// The date when the access token expires.
-@property(nonatomic, readonly, nullable) NSDate *expirationDate;
-/// The access token string.
-@property(nonatomic, copy, readonly, nullable) NSString *accessTokenString;
-/// The refresh token string.
-@property(nonatomic, copy, readonly, nullable) NSString *refreshTokenString;
+/// The access token object holding the string and expiration.
+@property(nonatomic, copy, readonly, nullable) GIDToken *accessToken;
+/// The refresh token object holding the string and no expiration.
```

Comment:
Nit: remove "no" so that we don't have to remember to update these docs if expiration is ever needed. Developers can infer no expiration if they get a token with nil in that property.

Reply:
Sounds good, done.